faw/pdf-observatory/faw_analysis_set.py

```diff
@@ -438,6 +440,7 @@ def update_status(status):
     # user downtime.
 
     if as_doc['status'] == AsStatus.REBUILD_IDS.value:
+        print('Rebuilding IDs...')
```

Comment:
I'd suggest removing these prints. Could wall them behind a global in the file. Also, if you keep them in a walled fashion, add the analysis set name.

Reply:
These have been extremely helpful for debugging, so I'd like to keep them, but yeah, they won't be necessary in prod. I'll set these up to be debug-level log lines, and set the logger's level via a top-level constant like you said (we could set the verbosity via a CLI arg in the future if there's a use for this sort of logging in other modules).